packages/xsnap/test/test-xs-perf.js

```diff
@@ -48,7 +48,7 @@ test('meter details', async t => {
 
   t.like(
     meters,
-    { compute: 1_380_192, allocate: 42_074_144 },
+    { compute: 1_380_187, allocate: 42_074_144 },
     'compute, allocate meters should be stable; update METER_TYPE?',
```

Comment:
I don't see a change to `METER_TYPE`. Did I miss it?

Reply:
Posting a commit.